playbooks/container-runtime/private/setup_storage.yml

```diff
@@ -13,7 +13,7 @@
         name: container_runtime
         tasks_from: docker_storage_setup_overlay.yml
       when:
-        - container_runtime_docker_storage_type|default('') == "overlay2"
```

Comment:
This var is used by `playbooks/aws/`, so these playbooks should be updated

cc @mazzystr